com.unity.cinemachine/Tests/Runtime/CinemachineBrainSetupTest.cs

```diff
@@ -0,0 +1,48 @@
+#if UNITY_EDITOR //Selection.activeGameObject
+using System.Collections;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Cinemachine.Tests.Editor
+{
+    [TestFixture]
+    public class CinemachineBrainSetup
```

Comment:
You should be inheriting CinemachineFixtureBase, which handles cleanup of objects.  Look at the other tests for examples.

Reply:
Done